scripts/5-print-versions.sh

```diff
@@ -26,6 +26,11 @@ SCRIPT_RUN=${SCRIPT_DIR}/.versions.run.sh
 SCRIPT_MOUNT=/.versions.sh
 CMD="bash ${SCRIPT_MOUNT}"
 
-echo "Getting version information from ${IMAGE_DEVEL} and ${IMAGE_CPP} ... "
-docker run --rm --gpus all -v ${SCRIPT_DEVEL}:${SCRIPT_MOUNT} ${IMAGE_DEVEL} ${CMD} | tee ${LOG_FILE}
-docker run --rm --gpus all -v ${SCRIPT_RUN}:${SCRIPT_MOUNT} ${IMAGE_CPP} ${CMD}| tee -a ${LOG_FILE}
+echo "Getting version information from ${IMAGE_DEVEL_ARCH} and ${IMAGE_CPP} ... "
+if [ "$ARCH" = "amd64" ]; then
+    docker run --rm --gpus all -v ${SCRIPT_DEVEL}:${SCRIPT_MOUNT} ${IMAGE_DEVEL_ARCH} ${CMD} | tee ${LOG_FILE}
+    docker run --rm --gpus all -v ${SCRIPT_RUN}:${SCRIPT_MOUNT} ${IMAGE_CPP} ${CMD}| tee -a ${LOG_FILE}
+elif [ "$ARCH" = "arm64" ]; then
```

Comment:
Same questions as with `4-test-libtensorflow-cc.sh`.

Reply:
Adjusted in [524ff24](https://github.com/ika-rwth-aachen/libtensorflow_cc/pull/2/commits/524ff24359332ab6b45889de9e69975e59c66675).